lib/modules/accountSwitcher.js

```diff
@@ -106,10 +120,18 @@ module.options = {
 	},
 };
 
-let userLink, $downArrow, $downArrowOverlay;
+let userLink, $downArrow, $downArrowOverlay, queryParameter;
 
 export let $accountMenu;
 
+module.beforeLoad = () => {
+	if(isPageType('comments') && module.options.autoSwitchOnPost.value) {
```

Comment:
It appears this patch won't let you deliberately switch to a different account. It would be great to check ["recently switched accounts on this post?"](https://github.com/honestbleeps/Reddit-Enhancement-Suite/pull/4443/files#r142846414) before even adding a hook to watchForThings.